### PR TITLE
Table comparisons of the web

### PR DESCRIPTION
**Pull request by Judeitopa**
Table describing web 1,  web 2, & web  3 in an easily understandable manner

### CODE REVIEW

**Comment by corwintines** [comment]
Thanks @Judeitopa!

There may be a use for a table here, but this would need a bit of work. Before doing that, id like to get a bit of input from others. 

@konopkja curious what you think about adding a table here in terms of UX?

@minimalsm given you created this page, wanted to see your thoughts on this?

**Comment by konopkja** [comment]
Comparison tables are great UX improvement for people who dont have time to read through the whole article in detail! I do think however that the explanations provided in this table could be improved with less jargon and a more descriptive approach. Once that is improved, I would love to see this on the page.

thank you @Judeitopa for taking action on this.

**Comment by Judeitopa** [comment]
@konopkja The table is actually a summary of the content of the page. I don't want long explanations on the table cos it has been done already. I will definately add the applications examples. Thanks!

**Comment by Judeitopa** [comment]
@konopkja Improved!. Let me know if you want something else added for further improvement. Thanks!

**Comment by konopkja** [comment]
> @konopkja Improved!. Let me know if you want something else added for further improvement. Thanks!

looks good from my side, just remove decentraland link in the table please. @corwintines @minimalsm opinions?

File: src/content/web3/index.md [hunk 1]
```diff
@@ -51,6 +51,15 @@ Although it's challenging to provide a rigid definition of what Web3 is, a few c
...
+ 
+ ##  TABLE : COMPARISONS OF THE EVOLUTION OF THE WEB ##
+ 
+  |WEB 1.0 | WEB 2.0 | WEB 3.0 |
+  | ------ | ------ | ------ |
+  | Permissionless | Permissioned | Permissionless |
+  | Static content | Dynamic content | Dynamic content |
+  | Does not require complex server-side architectures. | Companies run your agreements on their servers. | It is an ownership economy. You own part of the protocols you interact with. Your agreements are run on decentralized networks. |
+  | Online directories, Static informational websites(e.g news), Text-based online forums. | Social networking(e.g Twitter, Meta, LinkedIn), E-commerce(e.g Amazon, Shopify), Collaborative platforms. | Crypto wallets(e.g MetaMask), Decentralized exchanges(e.g Binance, Uniswap), Decentralized tokens(e.g Bitcoin, NFTs), Decentraland [see here](https://www.bing.com/ck/a?!&&p=0be7e71f4b178550JmltdHM9MTY3ODA2MDgwMCZpZ3VpZD0zOWM1Njk2Yi00MGY3LTZjNjYtMTRmYi03YjE1NDRmNzYyOGUmaW5zaWQ9NTE5MQ&ptn=3&hsh=3&fclid=39c5696b-40f7-6c66-14fb-7b1544f7628e&psq=decentraland&u=a1aHR0cHM6Ly9kZWNlbnRyYWxhbmQub3JnLw&ntb=1)|
```
Comment (wackerow) [inline src/content/web3/index.md:62]: Agree with @konopkja, would remove this Bing/Decentraland link. Would also suggest we highlight DEX's on Ethereum, and remove Binance from the examples.

Grammar notes: Need spaces between the list items and their examples, and `e.g.` should have two periods: "Social networking (e.g. Twitter, Meta, LinkedIn)".
Also, given our use of sentence casing throughout the site, would suggest only capitalizing the first word in each of these lists, unless it's a proper noun... not critical though.

**Comment by corwintines** [comment]
This seems to have been abandoned. @Judeitopa feel free to reopen with the above changes if I am mistaken.

**Pull request closed without merging**